vimiv/gui/metadatawidget.py

```diff
@@ -18,7 +18,7 @@
 _logger = utils.log.module_logger(__name__)
 
 
-if exif.piexif is not None:
+if exif.piexif is not None or exif.pyexiv2 is not None:
```

Comment:
Maybe we could add something like `exif.has_exif_support` which is currently simply `pyexiv2 is not None or piexif is not None` so we don't have to know about the library details elsewhere.

Reply:
I have actually considered replacing this check with something like `if exif.ExifHandler() != "":` That was actually the main reason why I have added the `filename=""` default. But in the end I did not add this because this check required deeper knowledge of the `ExifHandler` implementation.

Probably something as you suggest is most senseful.

Reply:
Looking good!